temba/orgs/models.py

```diff
@@ -86,6 +86,9 @@
 ACCOUNT_SID = 'ACCOUNT_SID'
 ACCOUNT_TOKEN = 'ACCOUNT_TOKEN'
 
+TWILIO_COMPLIANT_API_ACCOUNT_SID = 'TWILIO_COMPLIANT_API_ACCOUNT_SID'
+TWILIO_COMPLIANT_API_ACCOUNT_TOKEN = 'TWILIO_COMPLIANT_API_ACCOUNT_TOKEN'
```

Comment:
Out of curiosity, how far does the Twilio compatibility go? Does it include searching for available numbers for example? If not, then that definitely is another argument towards a more Kannel like approach, if it does, then that may indeed be more similar to the current Twilio approach.